src/pytest_mysql/exceptions.py

```diff
@@ -14,5 +14,9 @@ def __init__(self, output):
         super().__init__("Could not detect version in {}".format(output))
 
 
+class SocketPathTooLong(PytestMySQLException):
+    """Exception raised the socket path is over 103 chars (mysql will fail to start)."""
```

Comment:
there's a general pytest setting for tmpdir we should point to (`--basetemp`), while I think we could replicate this change from pytest-redis as well:
https://github.com/ClearcodeHQ/pytest-redis/pull/340